client/src/index.js

```diff
@@ -1,4 +1,30 @@
-import React from "react";
+import React, { Component } from "react";
 import ReactDOM from "react-dom";
 
-ReactDOM.render(<div>Hello</div>, document.getElementById("appMountPoint"));
+class ProxiedAPI extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { results: [] };
+  }
+
+  componentDidMount() {
+    fetch("/api/needsdiagnosis").then(res =>
```

Comment:
@magsout as long as we use fetch or xhr, I think it should work nicely.

Reply:
yes it is.